pyei/plot_utils.py

```diff
@@ -99,7 +141,7 @@ def plot_summary(
     sns.despine(ax=ax_box, left=True)
     # plot custom boxplot, with two boxplots in the same row
     colors = sns.color_palette()  # fetch seaborn default color palette
-    plot_props = dict(fliersize=5, linewidth=2, whis=[5, 95])
+    plot_props = dict(fliersize=5, linewidth=2, whis=[2.5, 97.5])
```

Comment:
This is the second "rider change" where I realized that the whiskers were showing a 90% credible interval, not a 95% credible interval!